master/internal/api_runs.go

```diff
@@ -139,6 +139,7 @@ func getRunsColumns(q *bun.SelectQuery) *bun.SelectQuery {
 		Column("r.external_run_id").
 		Column("r.project_id").
 		Column("r.searcher_metric_value").
+		ColumnExpr("r.archived").
```

Comment:
@keita-determined 
question: It looks like else where we are also checking the archived status the experiment & project, are we wanting to do the same with how we retrive it in this case?
cc: @AmanuelAaron

Reply:
This should be `"r.archived OR e.archived"`

Reply:
i found a potential bug.  
When i switch SQL with `r.archived OR e.archived AS archived`, the unarchive action does not unarchive experiment.archived. that causes this unexpected bug. this is not what we want, right?

https://github.com/determined-ai/determined/assets/109113616/fdb94e24-2ced-4921-afec-187ad08ca809